client/src/components/UserProfilePage.js

```diff
@@ -0,0 +1,36 @@
+import React, {useState, useEffect} from "react";
+import UserSaladList from "./UserSaladList"
+
+const UserProfilePage = (props) => {
+    const [user, setUser] = useState([])
+    const getUser = async() => {
+        try{
+            const response = await fetch("/api/v1/user-sessions/current")
+            if(!response.ok){
+                const errorMessage = `${response.status} (${response.statusText})`
+                const error = new Error(errorMessage)
+                throw(error)
+            }
+            const responseBody = await response.json()
+            setUser(responseBody)
+        } catch(error) {
+            console.error(`Error in Fetch: ${error.message}`)
+        }
+    }
+
+    useEffect(()=> {
+        getUser()
+    }, [])
```

Comment:
`App` is already fetching the currently signed in user. If we want this page to be specific for each user (a user can only see their Profile page), then the App BrowserRouter should render this route as an AuthenticatedRoute passing the current user

Reply:
Our thought process was that we would only fetch a user's reviews and posts when they access the profile page. If a user has a boatload of reviews and posts, all those items are fetched every time state changes are passed up to app.js if we tack those requests onto the base user request.